internalshared/configutil/encrypt_decrypt.go

```diff
@@ -0,0 +1,92 @@
+package configutil
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"regexp"
+
+	"github.com/gogo/protobuf/proto"
```

Comment:
Did you mean to use gogo here?

Reply:
Sure didn't. That stuff is insidious.  Will fix.